core/src/main/scala/org/apache/spark/sql/SnappyContext.scala

```diff
@@ -1009,8 +1009,7 @@ object SnappyContext extends Logging {
       }.orElse(Property.McastPort.getOption(conf).collectFirst {
         case s if s.toInt > 0 =>
           val url = "mcast-port=" + s
-          if (embedded) ExternalEmbeddedMode(sc, url)
-          else SplitClusterMode(sc, url)
+          /* if (embedded) */ ExternalEmbeddedMode(sc, url)
       }).orElse(Property.SnappyConnection.getOption(conf).collectFirst {
```

Comment:
Shouldn't this throw an exception like being done for locators option above saying that snappydata.connection property should be set? Also there is a typo in that exception message "paramater"